### ISSUE TITLE
No comprehensive tutorial

### ISSUE THREAD

**Issue by Vermeille** [cloudfoundry/uaa#661]
Hello,

I'm trying to understand if UAA could be used in my company, without CloudFoundry.

I've been trying to grasp the basics of UAA for days now.

I've been trying to use the sample UAA + App + API in order to get a practical approach. The only documentation I could find about it is that one: https://docs.cloudfoundry.org/concepts/architecture/uaa.html#demo-local . And it's totally not helpful. Good, I've made the first curl, and then cf-uaac is used, which is of zero use to me since I don't want a command line tool, but an API understanding so that I can integrate UAA with our services. And even without that tool, it stops when you got the token; nothing beside authenticating a pre-registered user has been done, and the tutorial stops here. I mean, really?

Here is a non exhaustive list of questions that are still left unanswered despite my ctrl+f, greps, and various other wanders and dabblings:

- Where is the user marissa created?
- Where are the apps API and App added to UAA? Their secrets key and various configs?
- Where is the configuration file for the sample code?
- How do I change the scopes marissa has access to? I can't use most of the API because that user doesn't have the needed accesses.
- What is the basic workflow to integrate another app / api / service with UAA so that it requires authentication?

Basically, the documentation needs a "Getting Started with UAA in 30 minutes", based for potential user scenarios.

Documentation like this ( https://docs.cloudfoundry.org/api/uaa/#authorization-code-grant ) is massively useless if you don't have a big picture understanding of UAA. So is the documentation in docs/ that covers only parts of various items and details.

For reference, https://getkong.org/docs/ is a user oriented documentation that allows you to get your basic needs fulfilled in less that an hour.

I'm willing to contribute that missing documentation if I finally can understand that product by whichever mean.

Best regards,

**Comment by sreetummidi** [comment]
@Vermeille We will be publishing a guide soon. Its under review.

**Comment by Vermeille** [comment]
Many thanks!

May I be part of (or spectate) the review process?

I'd be glad to contribute and an early access to this guide would be of immense value to me.

**Comment by prasadpete** [comment]
Is the documentation published yet please ?

**Comment by asc-media** [comment]
> We will be publishing a guide soon. Its under review.

That was more than a year ago - and I couldn't see anything like a guide????????

**Comment by githubuser88442** [comment]
Has there been an update on this?
